packages/core/src/index.ts

```diff
@@ -59,6 +59,8 @@ export {
 
 export {default as dynamic} from "next/dynamic"
 
+export {getConfig} from "next/config"
```

Comment:
I think this actually needs to be like this:

```suggestion
export {default as getConfig} from "next/config"
```

Sorry for leading you astray in slack!

Reply:
Thanks a lot for the correction. Let me fix it